explainer.md

```diff
@@ -70,20 +68,20 @@ Two examples where Anchors might update as real-world understanding improves are
 *   Anchors can lose tracking without application's intervention - there needs to be a way to notify the application of the fact that an anchor is no longer tracked by the system.
 
 
-## Add anchor - API Details
+## Anchor creation - API Details
+
+`XRFrame.createAnchor()` - free-floating anchor creation:
+- pose - initial pose where the anchor should be created - the system will make sure that the relationship with the physical world made at this moment in time is maintained as the tracking system's understanding of the world evolves.
+- referenceSpace - the frame of reference the pose is relative to.
 
-`addAnchor` parameters
-*   pose - the initial pose where the anchor should be created. The system will make sure that the relationship with the physical world made at this moment in time is maintained as the tracking system evolves.
-*   referenceSpace - the frame of reference the pose is relative to.
+The underlying system will attempt to keep the created anchor fixed relative to the real world.
 
-To enable feature detection of possible future versions of the API with additional parameters, an error is thrown if additional arguments are given to the function.
+`XRHitTestResult.createAnchor()` - attached anchor creation:
+- pose - initial pose where the anchor should be created - the system will make sure that the relationship with the physical object that caused this hit test result to be returned is maintained as the tracking system's understanding of the world evolves.
```

Comment:
This isn't clear.  What is the pose, relative to the hit test result?  Why not just the hit test result with no additional pose?

Reply:
Hit test result establishes a new coordinate system, the pose is expressed relative to it. This is to allow the applications to create anchors that are offset from a specific hit test. It may not be needed since this potentially could be maintained by the applications themselves.

Reply:
having the pose relative to the HitTestResult sounds good.